lib/src/views/Calendar/Day.tsx

```diff
@@ -123,6 +123,11 @@ export interface DayProps extends ExtendMui<ButtonBaseProps> {
    * @default false
    */
   disableHighlightToday?: boolean;
+  /**
+   * Allow selecting the same date (fire onChange even if same date is selected).
+   * @default false
```

Comment:
@eps1lon This change makes me wonder, on the main repository, we don't mention the default value in the TypeScript definitions, we have a tool that extracts them from the source. The upside here is that we get the information right in the editor:

<img width="801" alt="Capture d’écran 2020-07-14 à 20 28 03" src="https://user-images.githubusercontent.com/3165635/87462716-a9df9500-c610-11ea-8f58-bd8c5db7953c.png">

But it can get quickly out-of-sync. What do you think about the tradeoff?

Reply:
Great idea. I think we could statically verify if the default value matches. Though I wouldn't worry about it too much at first. In the end default values will rarely change in SemVer patches or minors because they're very likely breaking changes. 

I don't remember how we extract default values but if we extract them statically then we can also match them against TS types.